tests/Feature/AccountControllerTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Services\AccountService;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Mockery\MockInterface;
+use Tests\TestCase;
+
+
+class AccountControllerTest extends TestCase
+{
+    private array $wishlist;
+    private MockInterface $accountServiceSpy;
+    protected function setUp(): void {
+        parent::setUp();
+        $this->accountServiceSpy = $this->spy(AccountService::class);
+    }
+
+    //arrange
+    public function test_return_wishlist() {
```

Comment:
Looks good here! Should we add a test to check if the wishlist is passed an invalid I'd?

Reply:
good idea, yes eventually we want that. but since there is no logic in the service for invalid id, we can't really test that. also in this test we are mocking the accountService anyway, so we are only testing the controller.